pkg/runtime/interfaces/application_configuration.go

```diff
@@ -139,8 +139,9 @@ type NotificationsProcessorConfig struct {
 }
 
 type EmailServerConfig struct {
-	ServiceName  string `json:"serviceName"`
-	APIKeyEnvVar string `json:"apiKeyEnvVar"`
+	ServiceName    string `json:"serviceName"`
+	APIKeyEnvVar   string `json:"apiKeyEnvVar"`
```

Comment:
Mind adding a comment that one of APIKeyEnvVar or APIKeyFilePath should be specified?